tests/gis_tests/test_measure.py

```diff
@@ -294,6 +299,19 @@ def test_units_str(self):
         self.assertEqual(repr(a1), "Area(sq_m=100.0)")
         self.assertEqual(repr(a2), "Area(sq_km=3.5)")
 
+    def test_hectare(self):
+        a = A(sq_m=10000)
+        self.assertEqual(a.ha, 1)
+
+    def test_area_only_unit_att_name(self):
+        "Testing the `unit_attname` class method"
+        unit_tuple = [
+            ("Hectare", "ha"),
+        ]
+        for nm, att in unit_tuple:
+            with self.subTest(nm=nm):
+                self.assertEqual(att, A.unit_attname(nm))
```

Comment:
We have only one area-only unit, so we don't need a generic test:
```suggestion
        self.assertEqual(A.unit_attname("Hectare"), "ha")
```